src/ansys/fluent/core/workflow.py

```diff
@@ -366,6 +366,18 @@ def __dir__(self):
             )
         )
 
+    def get_state(self):
+        _state = self._task_container.get_state()
+        if "Workflow" in _state:
+            del _state["Workflow"]
+        returned_state = {}
+        for key, value in _state.items():
+            returned_state[str(key).replace("TaskObject:", "")] = value
```

Comment:
@prmukherj take care of the whole algorithm in one go and eliminate some of
the inaccuracies in the above implementation (we can discuss!)
```
for key, value in _state.items():
    type_and_name = key.split(":")
    if len(type_and_name) >=1 and type_and_name[0] == "TaskObject":
        returned_state[":".join(type_and_name[1:])] = value
```
BTW we should improve on the use of `replace` here. It is in